db/migrate/20191031022641_add_mock_interview_score.rb

```diff
@@ -0,0 +1,7 @@
+class AddMockInterviewScore < ActiveRecord::Migration
+  tag :predeploy
+  def change
+  	add_column :course71_project_scores, :total_score____mock_interview__feedback_, :integer
```

Comment:
Does the order of the columns need to match the order of the CSV export? If not, awesome. If so, I think you need to verify that the correct data is actually written to the correct column. E.g. go score one of the mock IV feedback assignments for a user and make sure that that value shows up in the exported data in the `total_score____mock_interview__feedback_` column. I wasn't sure if this was the case which is why the last time I fixed up this issue, I replaced the entire table instead of just adding missing columns: https://github.com/beyond-z/canvas-lms/pull/539/files

Reply:
It does need to show up in the right order. In this case, the additional column in the CSV is last, so it'll work out.